osx.txt

```diff
@@ -0,0 +1,18 @@
+# deps: brew install gcc@9 codec2 binutils
+[binaries]
+c       = 'gcc-9'
```

Comment:
As mentioned on discord, openrtx does not compile with macOS clang.

We coul try to make it compile, but in the end you still need to install gcc9 to compile miosix toolchain, so 🤷